python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -429,39 +432,16 @@ def _get_run_tags(self, run_id: str) -> Tags:
     def _get_command_args(self, run_args: ExecuteRunArgs, context: LaunchRunContext):
         return run_args.get_command_args()
 
-    def _get_image_for_run(self, context: LaunchRunContext) -> Optional[str]:
-        job_origin = check.not_none(context.job_code_origin)
-        return job_origin.repository_origin.container_image
+    @staticmethod
+    def get_image_for_run(run: DagsterRun) -> Optional[str]:
```

Comment:
still needed?

Reply:
I'd say yes for slightly better readability